test/test_schemes/initialize_constituents.F90

```diff
@@ -119,7 +119,6 @@ subroutine initialize_constituents_register(constituents, errmsg, errcode)
              vertical_dim = 'vertical_layer_dimension', &
              min_value = 0.0_kind_phys,                 &
              advected = .true.,                         &
-             water_species = .true.,                    &
```

Comment:
Maybe worth adding a comment stating why we aren't setting the `water_species` property here?  Otherwise I feel like future us will forget and add it back in again.

Reply:
Agreed @nusbaume I wasn't aware that `air_composition` could modify the water_species property after initialization so I thought it had to be set right here. A comment will be really helpful for when we forget about this one day :smile:

Reply:
comment added. thanks guys!